src/Microsoft.ML.PipelineInference/Interfaces/IPipelineOptimizer.cs

```diff
@@ -44,6 +45,7 @@ public abstract class PipelineOptimizerBase : IPipelineOptimizer
         protected IDataView OriginalData;
         protected IDataView FullyTransformedData;
         protected AutoInference.DependencyMap DependencyMapping;
+        protected Dictionary<string, ColumnPurpose> columnPurpose;
```

Comment:
>columnPurpose; [](start = 52, length = 14)

C# naming conventions, please.

Reply:
Fixed.

---
In reply to: [199662160](https://github.com/dotnet/machinelearning/pull/461#discussion_r199662160) [](ancestors = 199662160)